examples/regressions.ml

```diff
@@ -38,6 +38,7 @@ let compare name (old_ : CSet.t) (new_ : CSet.t) =
 let test new_l (name, old_l) =
   (* Cn is for unassigned code points, which are allowed to be
    * used in future version. *)
+  let old_l = CSet.to_list old_l in
```

Comment:
This line can be dropped, but you should also drop the lines below
```ocaml
let old_l =
      List.fold_left
        (fun acc (a, b) -> CSet.union acc (CSet.interval a b))
        CSet.empty old_l
    in
```

Reply:
Then to_list is no longer needed probably

Reply:
Addressed thanks!

Reply:
Actually this has been put back in place. I haven't looked at this code in a while and, at face value, the `to_list` call is just to work around a change in type declaration while removing the fold definitely is changing the code's runtime semantics so the safter choice seems to be to keep the new `to_list`.